Wikipedia/Code/NSDateFormatter+WMFExtensions.h

```diff
@@ -39,6 +39,6 @@
 
 + (instancetype)wmf_mediumDateFormatterWithoutTime;
 
-+ (instancetype)wmf_hyphenatedYearMonthDayFormatter;
++ (instancetype)wmf_englishHyphenatedYearMonthDayFormatter;
```

Comment:
@coreyfloyd sorry if there was some confusion about this again, but this formatter is only used to create page titles for network requests—i.e. never seen by the user.  I can separate these kinds of formatters out into a different category to clarify this.